### DISCUSSION TITLE
Ghostty

### DISCUSSION THREAD

**Discussion by tembbo** [catppuccin/catppuccin#2502] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

I haven't gotten the Ghostty beta yet, but i would love to see a port for it.

[Discord](https://discord.com/invite/Ghostty)

### Any additional comments?

You need to be in the closed beta.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
Already themed! Ghostty ships with themes from https://github.com/mbadolato/iTerm2-Color-Schemes: see https://github.com/mbadolato/iTerm2-Color-Schemes/blob/master/ghostty/catppuccin-frappe.

**Reply by uncenter** [reply]
Looks like the configuration format is a unique one, so I'll look into making an official port for it :)

**Reply by uncenter** [reply]
https://github.com/uncenter/ctp-ghostty

**Comment by tembbo** [comment]
Looks good, do you have access to the beta?

**Reply by uncenter** [reply]
Yep! The colors all look right from that master themes repository (I think they generate them based off of templates and a main data source), so you shouldn't reallly need to use https://github.com/uncenter/ctp-ghostty.

**Comment by sgoudham** [comment]
Closing as #2530 has been raised.
